portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+function validateForm() {
+  const sNumber = document.getElementById("number").value;
+  var errorSpan = document.getElementById("error-message");
+  const regex = /^\d+$/;
+  errorSpan.innerText = "";
+  if (sNumber.length > 0) {
+    if (regex.test(sNumber)) {
```

Comment:
Nice use of a regex!

Reply:
+1!

Reply:
Thanks!

Reply:
Found it strange that JS didn't have good built in string to int converting functions, so figured this would do the trick without converting within a try catch.